src/nhd_stream_tile_creation.sh

```diff
@@ -0,0 +1,5 @@
+#!/bin/bash
+
+#to create NHDPlus stream order tiles
+tippecanoe -zg --detect-shared-borders --simplify-only-low-zooms --drop-densest-as-need --simplification=5 --output-to-directory nhd_order_grouped nhdplus_order_1.geojson nhdplus_order_2_3.geojson nhdplus_order_4_5.geojson nhdplus_order_6.geojson
```

Comment:
This is in a Jenkins job now, right?

Reply:
yes, it got modified slightly to new geojsons but it's captured here more or less https://github.com/usgs-makerspace/wbeep-processing/blob/master/jenkins/create_rivers_basemap#L45